e2e-tests/cypress/tests/integration/channels/messaging/image_attachment_spec.js

```diff
@@ -97,7 +97,7 @@ describe('Image attachment', () => {
         // # File thumbnail should have correct dimensions
         verifyFileThumbnail({
             filename,
-            actualImage: {height: 151, width: 955},
+            actualImage: {height: 144, width: 906},
```

Comment:
why does this PR contain this change?

Reply:
Am unsure why that was included, I must have done something wrong when rebasing the branch for the update. 
My bad! Let me update it

Reply:
@M-ZubairAhmed just cleaned up commits.